<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>541</width>
    <height>333</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>541</width>
    <height>333</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>541</width>
    <height>333</height>
   </size>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::DefaultContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>PRC-506</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 0, 0);</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>541</width>
     <height>321</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="page_0">
    <widget class="QTextBrowser" name="textBrowser_boot">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>521</width>
       <height>311</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border-color: rgb(255, 255, 0);
border-style: solid;
border-width:2px;
color: rgb(255, 255, 0);</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_1">
    <widget class="QFrame" name="frame">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>521</width>
       <height>155</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-color: rgb(255, 255, 0);
border-style:solid;
border-width:2px;ui-&gt;textBrowser_boot-&gt;append(&quot;Welcome PRC-506 Radio...&quot;);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label_freq">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>50</y>
        <width>271</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>36</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border-color: rgb(0, 0, 0);
color: rgb(255, 255, 0);</string>
      </property>
      <property name="text">
       <string>878787877</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_unit">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>50</y>
        <width>91</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>36</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border-color: rgb(0, 0, 0);
color: rgb(255, 255, 0);</string>
      </property>
      <property name="text">
       <string>Hz</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_p">
      <property name="geometry">
       <rect>
        <x>3</x>
        <y>117</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border-color: rgb(0, 0, 0);
color: rgb(255, 255, 0);</string>
      </property>
      <property name="text">
       <string>PWR:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_power">
      <property name="geometry">
       <rect>
        <x>83</x>
        <y>117</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border-color: rgb(0, 0, 0);
color: rgb(255, 255, 0);</string>
      </property>
      <property name="text">
       <string>H</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_mod">
      <property name="geometry">
       <rect>
        <x>417</x>
        <y>117</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border-color: rgb(0, 0, 0);
color: rgb(255, 255, 0);</string>
      </property>
      <property name="text">
       <string>USB</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_ch_name">
      <property name="geometry">
       <rect>
        <x>187</x>
        <y>3</y>
        <width>331</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border-color: rgb(0, 0, 0);
color: rgb(255, 255, 0);</string>
      </property>
      <property name="text">
       <string>CH000</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_ch_index">
      <property name="geometry">
       <rect>
        <x>3</x>
        <y>3</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border-color: rgb(0, 0, 0);
color: rgb(255, 255, 0);</string>
      </property>
      <property name="text">
       <string>000</string>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="frame_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>521</width>
       <height>155</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-color: rgb(255, 255, 0);
border-style:solid;
border-width:2px;ui-&gt;textBrowser_boot-&gt;append(&quot;Welcome PRC-506 Radio...&quot;);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label_tune_state">
      <property name="geometry">
       <rect>
        <x>3</x>
        <y>120</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border-color: rgb(0, 0, 0);
color: rgb(255, 255, 0);</string>
      </property>
      <property name="text">
       <string>NOT TUNE</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_time">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>116</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 5px;
background-color: rgb(255, 255, 0);
color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>08:35:23</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_ale_broad">
      <property name="geometry">
       <rect>
        <x>3</x>
        <y>3</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border-color: rgb(0, 0, 0);
color: rgb(255, 255, 0);</string>
      </property>
      <property name="text">
       <string>BROADCAST</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_v">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>3</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border-color: rgb(0, 0, 0);
color: rgb(255, 255, 0);</string>
      </property>
      <property name="text">
       <string>VSWR:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_vswr">
      <property name="geometry">
       <rect>
        <x>407</x>
        <y>3</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border-color: rgb(0, 0, 0);
color: rgb(255, 255, 0);</string>
      </property>
      <property name="text">
       <string>0.0000</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_system_log">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>50</y>
        <width>441</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border-color: rgb(255, 0, 0);
color: rgb(255, 0, 0);
border-width:4px</string>
      </property>
      <property name="text">
       <string>Change Chanel Error</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
